utility.sh

```diff
@@ -15,4 +15,4 @@ fi
 LAUNCH_MOD="${Mod:-"${MOD_ID}"}"
 
 cd engine
```

Comment:
Not checking for failure here is a problem, but i'm putting this off to a future PR that will overhaul the way we manage the engine dependency.